prisma/schema.prisma

```diff
@@ -149,3 +151,34 @@ model DeliveryLogLine {
   createdAt DateTime    @default(now())
   updatedAt DateTime    @updatedAt
 }
+
+model Conversation {
+  id        Int         @id @default(autoincrement())
+  name      String
+  createdBy Int
+  messages  Message[]
+  createdAt DateTime    @default(now())
+  updatedAt DateTime    @updatedAt
+  users     UsersAndConversations[]
+}
+
+model Message {
+  id                Int           @id @default(autoincrement())
+  createdBy         String
+  content           String
+  userId            Int
+  user              User          @relation(fields: [userId], references: [id])
+  conversationId    Int
+  conversation      Conversation  @relation(fields: [conversationId], references: [id])
+  createdAt         DateTime      @default(now())
+  updatedAt         DateTime      @updatedAt
+}
+
+model UsersAndConversations {
```

Comment:
In the plan, this join table was going to be handled implicitly by Prisma.

Currently it's an explicit many:many, with a different name than 'ConversationMember'.

What's the reasoning here?

Reply:
We did this way so we can use query to do some filtering.

Reply:
Ok - remember, the team process is to update the plan and get it reviewed by a teacher :)